osu.Game.Tests/Visual/Online/TestSceneVotePill.cs

```diff
@@ -0,0 +1,71 @@
+﻿// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using System;
+using System.Collections.Generic;
+using NUnit.Framework;
+using osu.Framework.Graphics;
+using osu.Game.Overlays.Comments;
+using osu.Framework.Allocation;
+using osu.Game.Online.API.Requests.Responses;
+using osu.Game.Users;
+using osu.Framework.MathUtils;
+
+namespace osu.Game.Tests.Visual.Online
+{
+    [TestFixture]
+    public class TestSceneVotePill : OsuTestScene
+    {
+        public override IReadOnlyList<Type> RequiredTypes => new[]
+        {
+            typeof(VotePill)
+        };
+
+        private VotePill votePill;
+
+        [BackgroundDependencyLoader]
+        private void load()
+        {
+            var randomComment = new Comment
+            {
+                IsVoted = false,
+                UserId = 4444,
+                VotesCount = 2,
+            };
+
+            AddStep("Log in", () => API.LocalUser.Value = new User
```

Comment:
can you split these three tests up until `[Test]` methods with names explaining what they are testing?